docs/content/config/security/mail_crypt.md

```diff
@@ -57,10 +77,18 @@ Official Dovecot documentation: https://doc.dovecot.org/configuration_manual/mai
         . . .
     ```
 
-6. Start the container, monitor the logs for any errors, send yourself a message, and then confirm the file on disk is encrypted:
-    ```
-    [root@ip-XXXXXXXXXX ~]# cat -A /mnt/efs-us-west-2/maildata/awesomesite.com/me/cur/1623989305.M6v�z�@�� m}��,��9����B*�247.us-west-2.compute.inE��\Ck*�@7795,W=7947:2,
-    T�9�8t�6�� t���e�W��S   `�H��C�ڤ �yeY��XZ��^�d�/��+�A
-    ```
+5. Start the `docker-mailserver` container with `docker-compose up -d`.
+6. [Verify encryption is working](#verifying-encryption).
+
+## Verifying Encryption
+
+Once the container is running:
 
-This should be the minimum required for encryption of the mail while in storage.
+  1. Monitor the logs for any errors.
+  2. Send yourself a message.
+  3. Confirm the received email is encrypted on disk. Example using the `xxd` command on the host:
+      ```
+      [ec2-user@ip-xxx docker-mailserver]$ xxd /mnt/efs-us-west-2/maildata/mydomain.com/user1/cur/1626112488.M548380P8093.ip-xxx.us-west-2.compute.internal,S=128377,W=130314:2,S | grep CRYPTED
+      00000000: 4352 5950 5445 4403 0702 0000 0002 0000  CRYPTED.........
+      ```
+      If you don't see any output, then encryption is not working properly.
```

Comment:
This is better thanks :)

Do you feel that step 3 is sufficient for the reader to know how to use `xxd`?

I am unable to test this out myself at present, but is it saying that in the `maildata` dir for this account, there is a file called `1626112488.M548380P8093.ip-xxx.us-west-2.compute.internal,S=128377,W=130314:2,S`? The `,`, `=`, and `:` characters don't sound particularly filesystem friendly.. [it looks like](https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations) only `:` is going to be problematic on Windows and older macOS filesystems.

If this ~80 char value is a filename, we should document that it's incompatible with NTFS and exFAT, (_potentially macOS, the HFS+ entry isn't clear to me and an APFS entry is lacking_). Those users would need to use docker data volumes instead of bind mounts for non-unix filesystems?

Reply:
I have an opinion that users should know where their maildata directory is and be able to figure out how to find the latest email message to run the command against. The names of the files are not something I've chosen or have control over either -- it looks like the defaults for the software.

Reply:
> The names of the files are not something I've chosen or have control over either -- it looks like the defaults for the software.

Then the feature must have a clear warning that it's not compatible with Windows filesystems (NTFS and exFAT would be most common today). The warning would explain the incompatibility is with filesystems that do not support `:` character in file names.

You run macOS and can probably confirm `:` is working fine, I assume you're using APFS? Try create a file or folder name with `:` via Finder, just in case Docker is being naughty :stuck_out_tongue: